src/SFML/Graphics/Font.cpp

```diff
@@ -639,11 +656,38 @@ Glyph Font::loadGlyph(std::uint32_t codePoint, unsigned int characterSize, bool
         width += 2 * padding;
         height += 2 * padding;
 
-        // Get the glyphs page corresponding to the character size
-        Page& page = loadPage(characterSize);
+        // Get the pages list corresponding to the character size
+        PageList& pages = m_pageLists[characterSize];
 
-        // Find a good position for the new glyph into the texture
-        glyph.textureRect = findGlyphRect(page, {width, height});
+        // Find a page that can fit well the glyph
+        Page* page = NULL;
+        for (PageList::iterator it = pages.begin(); it != pages.end() && !page; ++it)
```

Comment:
`&& page != nullptr`

Reply:
As above, I don't consider this an improvement